index.html.in

```diff
@@ -91,9 +91,9 @@
                       </div>
                       <p data-text="tagline" class="tagline"></p>
                   </div>
-                  <div class="pull-right hidden-xs" id="divPullRightHidden">
+                  <div class="pull-right hidden-xs" id="applyingWidth">
```

Comment:
Hey @sushain97 This was the div i was referring to sorry that was my mistake stating the name i was willing to name. i couldn't name this div and the <i> tag below this.And also please tell me where to start my next issue at apertium-apy or apertium-hml-tools.

Reply:
> This was the div i was referring to sorry that was my mistake stating the name i was willing to name.

This div should definitely not be removed. It contains the localeSelector. Maybe something like `#localeSelectContainer`.

>  i couldn't name this div and the ' 'tag below this.

The one below could be `#primaryLocaleGlobe`.

> And also please tell me where to start my next issue at apertium-apy or apertium-hml-tools.

Neither. There are a number of pending issues with this PR in my comments.

Reply:
Ya done with all others but naming stuff is confusing me.Would keep PR showing all the changes.